source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,274 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+You can use a text search to search documents for a term or a phrase. A
+term is a sequence of characters that excludes whitespace characters. A
+phrase is a sequence of terms that with any number of whitespace
+characters. You can use this for searching through unstructured text,
+which is text without a predefined format such as resumes and
+course catalogs. 
+
+In MongoDB, you can run a text search on a string field. A text
+search retrieves documents that contain the term or phrase in the
+specified field. 
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+In the following examples, a student browses through courses to
+enroll in at their college. The documents in the ``junior_courses``
+collection represent the courses available for junior year students
+with their intended majors: 
+
+.. code-block:: json
+
+    { "_id": 1, "name": "Forensic science", "major": ["Forensic Science", "Criminal Justice"] }
+    { "_id": 2, "name": "Discrete Math in Computer Science", "major": ["Computer Science", "Mathematics"] }
+    { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+    { "_id": 4, "name": "Computer science Seminar", "major": ["Computer Science"] }
+
+Text Index
+----------
+
+You must create a **text index** using the ``Indexes`` builder before
```

Comment:
This sentence is worded such that using the Index builder is necessary, but it's not. You can create a text index using anything that converts to a BSON document. Instead, separate the "you must" to apply only to the text index and apply "you can" to the Indexes builder (as an optional way to create it).